src/routes/Register.js

```diff
@@ -0,0 +1,121 @@
+import React from "react";
+import { useFormik } from "formik";
+import {
+  Button,
+  Container,
+  TextField,
+  Typography,
+  Link,
+  Card,
+} from "@material-ui/core";
+import { formStyles } from '../FormStyles';
+import { registerValidationSchema } from '../FormValidation';
+
+const Register = () => {
+  const formik = useFormik({
+    initialValues: {
+      name: "",
+      email: "",
+      confirmEmail: "",
+      password: "",
+      confirmPassword: "",
+    },
+    validationSchema: registerValidationSchema,
+    onSubmit: (values) => {
```

Comment:
This is nit picky and we can decide as a group what we want to do, but I've gotten very used to no parens around single arg functions, unless we are destructuring/typing the value of that arg.
```suggestion
    onSubmit: values => {
```

Reply:
True.  Some backstory: I was trying to figure out how to redirect to a dashboard page after the submit button is clicked. I am thinking we might need isSubmitting or isValidating (or both) eventually.